### PR TITLE
[Intl] Document timezone support

### PR DESCRIPTION
**Pull request by c960657**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes (symfony/symfony#17636) |
| Applies to | 3.0+ |
| Fixed tickets | N/A |

### CODE REVIEW

**Comment by WouterJ** [comment]
:+1: 

After your code PR has been merged, a `.. versionadded:: 3.1` directive should be added indicating this function was introduced in Symfony 3.1 (or whatever version it'll be merged in).

File: components/intl.rst [hunk 1]
```diff
@@ -335,6 +335,27 @@ to the current default locale::
...
+Timezones
+~~~~~~~~~
+
+The translations of timezone names can be found in the timezone bundle::
+
+    use Symfony\Component\Intl\Intl;
+
+    \Locale::setDefault('en');
+
+    $locales = Intl::getTimezoneBundle()->getTimezoneNames();
```
Comment (jakzal) [inline components/intl.rst]: This variable doesn't store locales but time zone names. Same few lines below.

File: components/intl.rst [hunk 2]
```diff
@@ -335,6 +335,27 @@ to the current default locale::
...
+    // => array('Africa/Abidjan' => 'Abidjan', ...)
+
+    $locale = Intl::getLocaleBundle()->getTimezoneName('America/St_Barthelemy');
+    // => 'St. Barthélemy'
+
+All methods accept the translation locale as the last, optional parameter,
+which defaults to the current default locale::
+
+    $locales = Intl::getLocaleBundle()->getLocaleNames('de');
```
Comment (jakzal) [inline components/intl.rst]: `getLocaleNames()` is code here, but the example below shows timezones.

File: components/intl.rst (hunk 1 above)
Comment (c960657) [inline components/intl.rst]: Fixed.

File: components/intl.rst (hunk 2 above)
Comment (c960657) [inline components/intl.rst]: Fixed. This time I have actually run the example code :blush:

File: components/intl.rst [hunk 3]
```diff
@@ -335,6 +335,27 @@ to the current default locale::
     $currencies = Intl::getCurrencyBundle()->getCurrencyNames('de');
     // => array('AFN' => 'Afghanische Afghani', ...)
 
+Timezones
+~~~~~~~~~
+
```
Comment (HeahDude) [inline components/intl.rst]: version added ?

**Comment by c960657** [comment]
Closing because https://github.com/symfony/symfony/pull/17636 was rejected.

**Pull request closed without merging**